meta/main.yml

```diff
@@ -10,6 +10,10 @@ dependencies:
   - role: debops.golang
     tags: [ 'depend::golang', 'depend::golang:gitlab', 'depend-of::gitlab', 'type::dependency:hard' ]
 
+  - role: debops.nodejs
+    tags: [ 'depend::nodejs', 'depend::nodejs:gitlab', 'depend-of::gitlab', 'type::dependency:hard' ]
+    nodejs__upstream: True
```

Comment:
Using `nodejs__upstream` here is not idempotent. When you run the `debops.nodejs` as standalone after this role has been executed, the `debops.nodejs` role will not know that you wanted to enable upstream and things might break. In a more general sense, roles provide variables that are specifically meant to be used via dependent variables and roles know how to handle them if their parameters are changed or missing. The `nodejs__upstream` variable is not one of these, and it is meant to be configured via the inventory so that execution of the `debops.nodejs` role in different contexts (normal playbook, as a dependency, etc.) is consistent.

An alternative would be to write about it in the role documentation but that's still not present...

So, GitLab now requires NodeJS to work? Is a minimum version required or will GitLab work with NodeJS from a given OS release (Jessie, Xenial)?